theme/templates/includes/access-row.html

```diff
@@ -72,15 +72,15 @@
 
     <td class="user-actions">
         <i @click="undoAccess(user, index)"
-           v-if="user.can_undo && quotaHolder.id !== user.id"
+           v-if="user.can_undo && (user.user_type==='group' || (user.user_type === 'user' && quotaHolder.id !== user.id))"
             {# TODO: manually initialize tooltips of dynamically created items #}
            class="fa fa-undo" aria-hidden="true" data-toggle="tooltip"
            title="Undo Share"></i>
     </td>
 
     <td class="user-actions">
         <span @click="user.id === currentUser ? showDeleteSelfDialog(user, index) : removeAccess(user, index)"
-              v-if="!(user.access === 'owner' && hasOnlyOneOwner) && user.id !== quotaHolder.id && (selfAccessLevel === 'owner' || user.id === currentUser)"
+              v-if="!(user.access === 'owner' && hasOnlyOneOwner) && !(user.user_type === 'user' && user.id == quotaHolder.id) && (selfAccessLevel === 'owner' || user.id === currentUser)"
```

Comment:
`user.id !== quotaHolder.id`
Why this change?  Why are user id and quotaholder id being compared at all?  Can we remove that?

Reply:
same as above, I was assuming that we don't want to remove access for users who are also the quaotHolder for the resource. But I haven't looked into the quota system at all so perhaps this is irrelevant...

Reply:
Ok, I did not think of why quotaHolder id was important, thanks for pointing that out.  It make sense to me to check quotaHolder.id.